### PR TITLE
fixed nativeroll's player collapse & resize bug

### PR DESCRIPTION
**Pull request by n-randin**
@lannka 
@zhouyx
Help with merge this plz.

### CODE REVIEW

File: ads/nativeroll.js [hunk 1]
```diff
@@ -39,6 +39,9 @@ function initPlayer(data) {
     container: '#c',
     desiredOffset: 50,
     gid: data.gid,
+    onError: () => {
+      window.context.noContentAvailable();
```
Comment (lannka) [inline ads/nativeroll.js]: please pass down the `window` from caller (`global`)

Comment (n-randin) [reply ads/nativeroll.js]: done

Comment (n-randin) [reply ads/nativeroll.js]: Or i need something like "global.window.contex..."?

Comment (lannka) [reply ads/nativeroll.js]:
```
function initPlayer(global, data) {
  ...
  global.context.noContentAvailable();
}
```

**Comment by n-randin** [comment]
@lannka Hi, can you check this again? thx

File: ads/nativeroll.js [hunk 2]
```diff
@@ -39,6 +39,14 @@ function initPlayer(data) {
     container: '#c',
     desiredOffset: 50,
     gid: data.gid,
+    onError: () => {
+      global.context.noContentAvailable();
```
Comment (lannka) [inline ads/nativeroll.js]: you need pass `global` object as function param

Comment (n-randin) [reply ads/nativeroll.js]: ah, ok, did not immediately understand what you mean. Done this.

**Pull request merged**